api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java

```diff
@@ -913,13 +913,27 @@ private static String generateNotebookUserCode(
     String descriptiveComment =
         String.format(
             "# This query represents dataset \"%s\" for domain \"%s\" and was generated for %s",
-            dataSetName, domainAsString, cdrVersionName);
+            dataSetName, domainAsString, cdrVersion.getName());
+
+    String cdrVersionCheck;
     String sqlSection;
     String dataFrameSection;
     String displayHeadSection;
 
     switch (kernelTypeEnum) {
       case PYTHON:
+        cdrVersionCheck =
+            String.format(
+                "notebook_cdr = '%s.%s'\n"
```

Comment:
BigQuery Project and Dataset associated with the Workspace at the time of Notebook creation

Reply:
Open to alternative names.  creation_time_cdr?